static/app/components/comboBox/index.tsx

```diff
@@ -242,15 +242,17 @@ function ControlledComboBox<Value extends string>({
 
   // Sync input value with value prop
   const previousValue = useRef(value);
-  if (previousValue.current !== value) {
-    const selectedLabel = options
-      .flatMap(item => ('options' in item ? item.options : [item]))
-      .find(option => option.value === value)?.label;
-    if (selectedLabel) {
-      setInputValue(selectedLabel);
+  useEffect(() => {
```

Comment:
did we decide to use `useEffect` because the change of the values is a side effect?

Reply:
Yes, value comes from `options` list, and it should be updated if that list is updated. And that list might get updated if the screen size changes